lib/java_buildpack/container/tomcat/tomcat_geode_store.rb

```diff
@@ -44,8 +44,10 @@ def release
         credentials = @application.services.find_service(FILTER, KEY_LOCATORS, KEY_USERS)['credentials']
         user = credentials[KEY_USERS].find { |u| cluster_operator?(u) }
 
-        @droplet.java_opts.add_system_property 'gemfire.security-username', user['username']
-        @droplet.java_opts.add_system_property 'gemfire.security-password', user['password']
+        File.open(@droplet.root + 'gemfire.properties', 'w') do |f|
+          f.write('security-username=' + user['username'] + "\nsecurity-password=" + user['password'])
+        end
```

Comment:
Now that we are modify the filesystem I think we should move this logic from `release` to `compile`. The organization is described in these comment on `JavaBuildpack::Component::BaseComponent`

https://github.com/cloudfoundry/java-buildpack/blob/main/lib/java_buildpack/component/base_component.rb#L58-L79

While we are at it we should probably move this into a a private methods called `create_gemfire_properties` (writes file) `gemfire_properties_path`(returns path) and `gemfire_properties`(returns filename), mirroring the organization of the `create_client_cache` etc.

Reply:
Ah, ok, good suggestion!

Reply:
Deleted the addition of this file altogether.